rosdep/python.yaml

```diff
@@ -7194,6 +7194,19 @@ python3-yaml:
       packages: [pyyaml]
   rhel: ['python%{python3_pkgversion}-PyYAML']
   ubuntu: [python3-yaml]
+python3-yappi:
+  debian:
+    '*': [python3-yappi]
+    buster: null
+    stretch: null
+  fedora: [python3-yappi]
+  gentoo: [dev-python/yappi]
+  osx:
+    pip: [yappi]
```

Comment:
Should this live in a separate `python3-yappi-pip` rule? @tfoote @nuclearsandwich please advise.

Reply:
No, if you're filling in a platform with a pip rule but it's available from apt as per usual that's correct. The `-pip` extension is for things that are exclusively pip provided and might collide with a non-pip rule in the future.